src/Hazelcast.Net/Exceptions/DistributedObjectDestroyedException.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace Hazelcast.Exceptions
+{
+    internal class DistributedObjectDestroyedException : HazelcastException
```

Comment:
we don't want to introduce specialized remote exceptions for now.

Reply:
So, should I just throw `RemoteException` ? Most of the tests look for `DistributedObjectDestroyedException` whether is thrown.